WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/HistoryItemViewModelBase.cs

```diff
@@ -87,6 +87,9 @@ protected string GetConfirmedToolTip(int confirmations)
 
 	public bool IsCPFPd { get; set; }
 
+	public bool IsIncomingTransactionDisplayed => IsConfirmed && (IncomingAmount != null && IncomingAmount > Money.Zero);
+	public bool IsOutgoingTransactionDisplayed => IsConfirmed && (OutgoingAmount != null && OutgoingAmount > Money.Zero);
```

Comment:
Checking null here is not needed, right?

```suggestion
	public bool IsIncomingTransactionDisplayed => IsConfirmed && IncomingAmount > Money.Zero;
	public bool IsOutgoingTransactionDisplayed => IsConfirmed && OutgoingAmount > Money.Zero;
```

Reply:
It is, actually. Incoming and Outgoing Amounts are nullable properties.

Reply:
AFAIU if `IncomingAmount` is `null` then `IncomingAmount > Money.Zero` would return false and thus we don't really need to check `IncomingAmount`.

Reply:
AFAIU if `IncomingAmount` is `null` then `IncomingAmount > Money.Zero` would return false and thus we don't really need to check `IncomingAmount`.